{
    "title" : "FailedToImpersonateUsersException",
    "uri" : "http://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
    "printableUri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
    "clickUri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
    "uniqueId" : "42.4247$http://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
    "excerpt" : "FailedToImpersonateUsersException ... FailedToImpersonateUsersException : There was no certificate provided by the client while trying to impersonate users. ... Either provide a trusted client cert...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1030,
    "percentScore" : 72.21627,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "FailedToImpersonateUsersException",
      "sysauthor" : "deepak_raghav",
      "sysurihash" : "PjZEXr9NR20Hi5Dg",
      "urihash" : "PjZEXr9NR20Hi5Dg",
      "numberofanswers" : 1,
      "sysuri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
      "sysprintableuri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
      "sitemappriority" : 0.1,
      "systransactionid" : 160349,
      "questionaskeddatestring" : "·\n            Jan 27, 2015 at 02:32 AM ·",
      "tags" : [ "coveoforsitecore" ],
      "sysconcepts" : "Coveo ; credentials ; login ; client ; certificate ; FailedToImpersonateUsersException ; try acessing ; dev server ; impersonation rights",
      "concepts" : "Coveo ; credentials ; login ; client ; certificate ; FailedToImpersonateUsersException ; try acessing ; dev server ; impersonation rights",
      "printableuri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
      "documenttype" : "WebPage",
      "sysindexeddate" : 1517941923000,
      "permanentid" : "cedbd83235d03a2d9dfe5289b73854851248757f9bae78724938e187f848",
      "syslanguage" : [ "English" ],
      "transactionid" : 160349,
      "title" : "FailedToImpersonateUsersException",
      "date" : 1422316800000,
      "objecttype" : "Answers",
      "sitemapchangefrequency" : "Monthly",
      "answerauthor" : [ "Vincent Séguin" ],
      "hasanswer" : "true",
      "audience" : [ "Administrator", "Developer" ],
      "hasacceptedanswer" : "false",
      "sourcetype" : "Sitemap",
      "sysconnectortype" : "SitemapCrawler",
      "rowid" : 5447317,
      "numberofcomments" : 0,
      "size" : 69559,
      "sysdocumenttype" : "WebPage",
      "questionvotes" : 0,
      "clickableuri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
      "syssource" : "AnswersCloud",
      "orderingid" : 212384744911577647,
      "acceptedanswers" : 0,
      "syssize" : 69559,
      "sysdate" : 1422316800000,
      "author" : "deepak_raghav",
      "source" : "AnswersCloud",
      "questionbodyhtml" : "<div class=\"question-body\">\n                <p>FailedToImpersonateUsersException : There was no certificate provided by the client while trying to impersonate users. Either provide a trusted client certificate with impersonation rights or login with requested credentials.<strong>strong text</strong></p> \n<p>I have Coveo installed using the service credentials in my dev server and when i login using my log on cred and try acessing the Rest end point this error is coming.</p> \n<p>I seen other posts getting redirected for Coveo Support for the same issue. Has it been fixed or something should be done from my end as well.</p>\n                </div>",
      "sitemaplastmodified" : 1422316800000,
      "collection" : "default",
      "syssourcetype" : "Sitemap",
      "indexeddate" : 1517941923000,
      "questionnumber" : 3789,
      "connectortype" : "SitemapCrawler",
      "filetype" : "html",
      "sysclickableuri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
      "sysfiletype" : "html",
      "language" : [ "English" ],
      "sitemapparenturl" : "http://answers.coveo.com/questions/1.xml",
      "questionauthor" : "deepak_raghav",
      "sitename" : "Answers",
      "sysrowid" : 5447317,
      "uri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
      "syscollection" : "default",
      "sitelanguage" : [ "English" ]
    },
    "Title" : "FailedToImpersonateUsersException",
    "Uri" : "http://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
    "PrintableUri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
    "ClickUri" : "https://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
    "UniqueId" : "42.4247$http://answers.coveo.com/questions/3789/failedtoimpersonateusersexception.html",
    "Excerpt" : "FailedToImpersonateUsersException ... FailedToImpersonateUsersException : There was no certificate provided by the client while trying to impersonate users. ... Either provide a trusted client cert...",
    "FirstSentences" : null
  }